.eslintrc.json

```diff
@@ -21,7 +24,10 @@
                     }
                   ]
                 }
-              ]
+              ],
+            "@typescript-eslint/naming-convention": "off",
+            "jsdoc/check-tag-names": "off",
+            "@typescript-eslint/default-param-last": "off"
```

Comment:
Rules like `@typescript-eslint/default-param-last` are good for code readability. Could we consider setting it as `warn`?

Reply:
Yes either that, but depends on how many instances there are. If not many better to turn off on specific lines.